### PR TITLE
Add refresh_patreon command to rotate access token

### PR DESCRIPTION
**Pull request by MrTeferi**
# Description

- Command refreshes the access and refresh tokens, then updates the .env accordingly
- Added script for use with crontab
- Updated env to hold all relevant Patreon keys
- Fixed broken isort pre-commit config
- Downgraded django-crispy-forms to 1.14.0 pending migration to 2.0

# Checklist

- [x] I have installed `pre-commit` and run the hooks with `pre-commit run`.
- [x] I have updated any related tests for code I modified or added new tests where appropriate.
- [x] I have updated any relevant documentation or created new documentation where appropriate.

### CODE REVIEW

**Comment by MrTeferi** [comment]
Went ahead and rolled in the proposed fix for issue #110 so we can close PR #120. I haven't kept up with Docker so I don't actually know if this will fix the problem, but I do know that env variable is indeed supposed to be in there, so can't hurt to add it :)

File: MPCAutofill/requirements.txt [hunk 1]
```diff
@@ -2,8 +2,9 @@ attrs
 chardet
 click==8.0.4
 defusedxml
+python-dotenv
 Django~=4.0
-django-crispy-forms
+django-crispy-forms==1.14.0
```
Comment (ndepaola) [inline MPCAutofill/requirements.txt:7]:
```suggestion
django-crispy-forms~=1.1
```

File: MPCAutofill/MPCAutofill/.env.dist [hunk 2]
```diff
@@ -13,6 +13,7 @@ PREPEND_WWW=False
 
 # Postgres Database, create username/pass for security
 DATABASE_ENGINE=django.db.backends.postgresql
+DATABASE_NAME=mpcautofill
```
Comment (ndepaola) [inline MPCAutofill/MPCAutofill/.env.dist:16]: i merged the PR you linked into master, could you back this change out of this PR?

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/pycqa/isort
-    rev: 5.9.0
+    rev: 5.11.5
```
Comment (ndepaola) [inline .pre-commit-config.yaml:3]: i also fixed this as part of some repo maintenance today, could you back this change out of this PR?

Comment (MrTeferi) [reply .pre-commit-config.yaml:3]: Done!

File: MPCAutofill/MPCAutofill/.env.dist (hunk 2 above)
Comment (MrTeferi) [reply MPCAutofill/MPCAutofill/.env.dist]: Done!

**Pull request closed without merging**